packages/odata-common/src/request-builder/batch/batch-request-builder.ts

```diff
@@ -33,8 +34,9 @@ export class BatchRequestBuilder<
     readonly defaultServicePath: string,
     readonly requests: (
       | BatchChangeSet<DeSerializersT>
-      | GetAllRequestBuilderBase<EntityBase, DeSerializersT>
-      | GetByKeyRequestBuilderBase<EntityBase, DeSerializersT>
+      | Omit<GetAllRequestBuilderBase<EntityBase, DeSerializersT>, 'execute'>
+      | Omit<GetByKeyRequestBuilderBase<EntityBase, DeSerializersT>, 'execute'>
+      | Omit<ActionFunctionImportRequestBuilderBase<any, any>, 'execute'>
```

Comment:
I guess, because we don't know the parameters/return types here.

Reply:
yes that is the reason. I changed it to `unknown`for the return type, because we really do not have any base type and for the request config I used the base of the Action/Function Imports  which is the `ODataRequestConfig`